### ISSUE TITLE
Administrate does not yet support namespaced models.

### ISSUE THREAD

**Issue by AfolabiOlaoluwa** [thoughtbot/administrate#1026]
Am working on a Modular Application by following TutsPlus Modular Rails App training [https://github.com/mmuehlberger/tutsplus-modular-rails-app](url). 

However, after install Administrate Gem and running `rails generate administrate:install` to install my dashboard, I get the error as is in the image below:

<img width="479" alt="screen shot 2017-11-15 at 09 23 01" src="https://user-images.githubusercontent.com/15254043/32826053-403f1580-c9e7-11e7-966c-9e5638cf75eb.png">

In short Administrate will not work in a Modular Rails App. How do I fix this problem?

**Comment by nickcharlton** [comment]
We've been working on supporting namespaced models for the past few months and now broadly it should be working.

I would recommend ignoring the warning and going ahead with your project, but it'd be great if you could raise any issues you come across!

I've been meaning to evaluate how good our support is, and whether or not the warning could now go.

**Comment by AfolabiOlaoluwa** [comment]
Yes I figured out I could proceed and ignore the warning, but Rubymine somewhat shows it can’t be used with its dependency underlined in the generated scaffold/installation.

**Comment by AfolabiOlaoluwa** [comment]
I will proceed as you advices and will post any error I see or a pull request on how I solved it.

**Comment by nickcharlton** [comment]
Thanks! I'm going to close this for now, so please open new specific issues if you come across them!

**Comment by sedubois** [comment]
I was also put off by these warnings, but with patience apparently managed to get my PaperTrail and Globalize namespaced models hooked up:
https://github.com/thoughtbot/administrate/issues/1291#issuecomment-545054789
